src/System Application/App/Permission Sets/src/PermissionSetRelationImpl.Codeunit.al

```diff
@@ -364,6 +398,22 @@ codeunit 9856 "Permission Set Relation Impl."
         end;
     end;
 
+    local procedure GetPermissionSetName(AppId: Guid; RoleId: Code[30]; Scope: Option System,Tenant): Text[30];
```

Comment:
Minor: Could we make the scope option an enum? Not just for permission set, but in general. After all it's used everywhere: System, Tenant.

Reply:
This could be done in a separate PR for the entire module, if we want to prioritize that. This is how it's done currently in the entire module and the permission set tables themselves are also using an option.